### ISSUE TITLE
"Git pull" feature for sysadmins

### ISSUE THREAD

**Issue by petterroea** [InfectedLan/InfectedAPI#165]
This would have a button on the crewpage, but i imagine it being handled by the api, so i'l make a ticket here.

A button that automatically pulls newest changes from git sounds awesome. It would require server-specific files(like htaccess and settings) to be stored somewhere else and copied in when the pull is done, though.

Still, a wish.

**Comment by halvors** [comment]
Yeah. But this would be test specific. Is this i'll rather have it auto updating the test server.

**Comment by halvors** [comment]
@petterroea Do you agree with this?

**Comment by petterroea** [comment]
Do you mean that anything put on master is automatically deployed?

**Comment by halvors** [comment]
@petterroea What's the status on this one?

**Comment by petterroea** [comment]
Doing it once i have a working docker setup for local dev. Pri 1.
